### DISCUSSION TITLE
Struggling with media storage size

### DISCUSSION THREAD

**Discussion by viralpoet** [mastodon/mastodon#27599] [General]
I am running mastodon 4.2.0 the membership on my instance is fairly static. But in the last month my S3 storage has ballooned.
These reported sizes I’m happy with 
![IMG_0340](https://github.com/mastodon/mastodon/assets/17892651/62db4dcf-cdce-450f-9db3-6b4225ddd453)

But on my S3 storage it’s showing 200gb and 600,000 files
![IMG_0339](https://github.com/mastodon/mastodon/assets/17892651/8bae6773-86ba-4af1-8227-19eac29abbd7)

I occasionally run media remove to delete attachments in the cache. I been doing a series of commands using

RAILS_ENV=production bin/tootctl media remove --days 5

It runs for a few minutes and gets killed. The number of files to process does reduce to zero but the S3 server space appears to remain the same. What’s going on?

![IMG_0341](https://github.com/mastodon/mastodon/assets/17892651/58f57200-ff67-4a00-b0d4-ed536ecdd130)
![IMG_0342](https://github.com/mastodon/mastodon/assets/17892651/18bd9df4-1aec-4a8f-b4bd-9da64c86c184)

**Comment by vmstan** [comment]
Is it possible the process runs out of resources?
